services/demarchesSimplifiees/autoAccept.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path');
+const graphql = require('../../utils/graphql');
+const config = require('../../utils/config');
+const { DOSSIER_STATE } = require('../../utils/dossierState');
+const {
+  default: {
+    getChampsIdFromField,
+    getAnnotationsIdFromField,
+  },
+} = require('../champsAndAnnotations');
+const { getAllPsychologistList } = require('./demarchesSimplifiees');
+const { default: uploadDocument } = require('./uploadDocument');
+
+const FILE = path.join(__dirname, '..', '..', 'static', 'documents', 'parcours_psychologue_sante_psy_etudiant.pdf');
+
+const sendAutoAcceptMessage = async (dossierId) => {
+  const uploadFileId = await uploadDocument(FILE, dossierId);
```

Comment:
So sad to upload the same document for every dossier.. Is there a way to upload it at the demarche level and use same file for all dossiers?

Reply:
no... it has to be related to a dossierId :/